packages/lexical-playground/src/plugins/ToolbarPlugin/fontSize.tsx

```diff
@@ -23,13 +23,21 @@ import {
 } from './utils';
 
 export function parseAllowedFontSize(input: string): string {
-  const match = input.match(/^(\d+(?:\.\d+)?)px$/);
-  if (match) {
-    const n = Number(match[1]);
-    if (n >= MIN_ALLOWED_FONT_SIZE && n <= MAX_ALLOWED_FONT_SIZE) {
-      return input;
-    }
+  const pxMatch = input.match(/^(\d+(?:\.\d+)?)px$/);
+  const ptMatch = input.match(/^(\d+(?:\.\d+)?)pt$/);
+
+  let pxValue: number | undefined = pxMatch ? Number(pxMatch[1]) : undefined;
+
+  if (ptMatch) {
+    pxValue = Number(ptMatch[1]) * (4 / 3);
+  } else {
+    return '';
   }
+
+  if (pxValue >= MIN_ALLOWED_FONT_SIZE && pxValue <= MAX_ALLOWED_FONT_SIZE) {
+    return input;
+  }
+
```

Comment:
I think this is wrong because either pxMatch or ptMatch will be truthy, but this will return '' whenever ptMatch is not  truthy. I'm surprised we don't have any test coverage for that, so I think we should add a test that makes sure that the px case is also tested.

```suggestion
  const pxptMatch = input.match(/^(\d+(?:\.\d+)?)(px|pt)$/);
  if (!pxptMatch) {
    return '';
  }

  const pxValue = Number(pxptMatch[1]) * (pxptMatch[2] === 'pt' ? 4 / 3 : 1);
  if (pxValue >= MIN_ALLOWED_FONT_SIZE && pxValue <= MAX_ALLOWED_FONT_SIZE) {
    return input;
  }

```